Champions/Karthus/Q.cs

```diff
@@ -0,0 +1,72 @@
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Spells
+{
+    public class KarthusLayWasteA1 : IGameScript
+    {
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            AddParticleTarget(owner, "Karthus_Base_Q_Hand_Glow.troy", owner, 1, "R_Hand");
+            AddParticle(owner, "Karthus_Base_Q_Point.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Ring.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Skull_Child.troy", spell.X, spell.Y);
+            CreateTimer(0.5f, () =>
+            {
+                IGameObject m = AddMinion(owner, "TestCube", "TestCube", spell.X, spell.Y);
+                var range = GetUnitsInRange(m, 100, true);
+                var ap = owner.Stats.AbilityPower.Total;
+                var damage = 20f + spell.Level * 20f + ap * 0.3f;
+                
+                foreach (var units in range)

```

Comment:
Yes, this looks cool, but for an outsider reading lines below, it is confusing. Please rename these variables for readability.